### PR TITLE
refactor(website) Added simple spec

### PR DESCRIPTION
**Pull request by 9renpoto**
# TL;DR

Added Top page

screenshot-2018-3-14 userdive knowing it makes the world better

### CODE REVIEW

File: website/README.md [hunk 1]
```diff
@@ -1,3 +1,21 @@
-# @userdive/agent documents
+# @userdive/developers document
```
Comment (Surgo) [inline website/README.md]: developer documentation?

File: website/core/Footer.js [hunk 2]
```diff
@@ -14,8 +15,21 @@ module.exports = function Footer ({
             height='58'
           />
         </a>
+        <div>
+          <h5>Docs</h5>
+          <a href={`${baseUrl}docs/${language}/getting-started.html`}>Usage</a>
+          <a href={`${baseUrl}docs/${language}/plugins.html`}>Plugins</a>
+          <a href={`${baseUrl}docs/${language}/integrations.html`}>
+            Integrations
```
Comment (Surgo) [inline website/core/Footer.js]: no line break in inline tag

Comment (Surgo) [reply website/core/Footer.js]: oh removed it by minify

File: website/pages/en/index.js [hunk 3]
```diff
@@ -138,22 +118,18 @@ const Showcase = ({ language }) => {
...
-    })
+  const showcase = users.filter(({ pinned }) => pinned).map((user, i) => (
+    <a href={user.infoLink} key={i}>
+      <img src={user.image} title={user.caption} />
+    </a>
+  ))
 
   return (
     <div className='productShowcaseSection paddingBottom'>
       <h2>{"Who's Using This?"}</h2>
```
Comment (Surgo) [inline website/pages/en/index.js]: `Customer Success` ?

File: website/pages/en/index.js [hunk 4]
```diff
@@ -67,68 +71,44 @@ const Block = ({ id, background, children, layout }) => (
...
 )
 
-const LearnHow = props => (
+const Migrate = props => (
   <Block background='dark'>
     {[
       {
-        content: 'Talk about learning how to use this',
-        image: imgUrl('docusaurus.svg'),
+        content: 'To next generation USERIDVE',
```
Comment (Surgo) [inline website/pages/en/index.js]: `next generation` 🤔

File: .travis.yml [hunk 5]
```diff
@@ -17,7 +17,7 @@ jobs:
       script: npm run ci:sauce
       if: type = pr or branch = master
     - stage: deploy
-      script: skip     # usually you do not want to rerun any tests
+      script: npm run build
```
Comment (9renpoto) [inline .travis.yml]: あー、これだと結局動くのかあ

**Pull request merged**